AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -7320,7 +7320,7 @@ biedronka.pl###cookiepopupdiv
 slidedog.com,finanztreff.de###cookiereminder
 lamarinaresort.com###cookies-advice
 roxy-germany.de,quiksilver.de,billabong.de,uk.gilera.com###cookies-disclaimer
-hitpaw.com,tenorshare.com,miuios.cz,ascii-code.com,htmlsymbols.xyz,dietotenhosen.de,cambio16.com,hacker-pschorr.com,gamersglobal.de,spinninrecords.com,operamusica.com,splashdamage.com###cookies-eu-banner
+hitpaw.com,tenorshare.com,miuios.cz,ascii-code.com,htmlsymbols.xyz,dietotenhosen.de,cambio16.com,hacker-gowork.pl,pschorr.com,gamersglobal.de,spinninrecords.com,operamusica.com,splashdamage.com###cookies-eu-banner
```

Comment:
Please revert `hacker-pschorr.com`.
`hacker-gowork.pl` it should be at the beginning of the rule as `gowork.pl,`.

Reply:
You meant force push to avoid see commit in history?

Reply:
I meant this [`f3e69c7` (#141185)](https://github.com/AdguardTeam/AdguardFilters/pull/141185/commits/f3e69c7a2aaa9f8767396ba36121829ace9e7fd6).